titan-server/src/schema.rs

```diff
@@ -222,6 +222,21 @@ table! {
     }
 }
 
+table! {
+    user_event_excuses (id) {
```

Comment:
Refer to the above comment in the SQL migration, but anything that isn't wrapped in `Nullable<T>` in diesel's schema should be marked `NOT NULL` in the actual SQL.